deployment/administration/SRE_Delete_Unassigned_Users.ps1

```diff
@@ -0,0 +1,25 @@
+param(
+    [Parameter(Mandatory = $true, HelpMessage = "Enter SHM ID (e.g. use 'testa' for Turing Development Safe Haven A)")]
+    [string]$shmId
+)
+
+Import-Module Az.Accounts -ErrorAction Stop
+Import-Module $PSScriptRoot/../common/AzureCompute -Force -ErrorAction Stop
+Import-Module $PSScriptRoot/../common/Configuration -Force -ErrorAction Stop
+Import-Module $PSScriptRoot/../common/Logging -Force -ErrorAction Stop
+
+# Get config
+# -------------------------------
+$config = Get-ShmConfig -shmId $shmId
+$originalContext = Get-AzContext
+
+# Delete users not currently in a security group
+# ----------------------------------------------
+$null = Set-AzContext -SubscriptionId $config.subscriptionName -ErrorAction Stop
+Add-LogMessage -Level Info "Deleting users not assigned to any security group: $($config.shm.id) from $($config.dc.vmName)..."
+
+$script = "Delete_Unassigned_Users.ps1"
```

Comment:
Currently all the scripts in `administration` are ones that can be run locally. Can you put `Delete_Unassigned_Users.ps1` somewhere else (e.g. in a `remote` subdirectory of `administration`)?